pep517/wrappers.py

```diff
@@ -35,6 +35,10 @@ def __init__(self, backend_name, backend_path, message):
         self.message = message
 
 
+class HookMissing(Exception):
+    """Will be raised on missing hooks."""
```

Comment:
I'd like the exception to carry the name of the missing hook, so that if it's uncaught and reaches the user, it's clear what it was looking for.